universal-login-react/src/ui/ConnectionFlow/ChooseConnectionMethod.tsx

```diff
@@ -15,34 +17,33 @@ interface ChooseConnectionMethodProps {
 export const ChooseConnectionMethod = ({onCancel, onConnectWithDeviceClick, onConnectWithPassphraseClick, className}: ChooseConnectionMethodProps) => {
   const ref = useRef<HTMLDivElement>(null);
   const [progress, setProgress] = useState(0.0);
+  const theme = useThemeName();
 
   return (
-    <div className="universal-login-connection">
-      <div className={getStyleForTopLevelComponent(className)}>
-        <div className="connection">
-          <h1 className="connection-title">Connect with another device</h1>
-          <div className="connection-content">
-            <div className="connection-row-wrapper" ref={ref} onScroll={() => calculateScrollProgress(ref, setProgress)} >
-              <div className="connection-row">
-                <div className="connection-method passwordless-method">
-                  <h2 className="connection-method-title">Passwordless</h2>
-                  <p className="connection-method-text">Approve the connection with another device that already controls your account.</p>
-                  <button id="emoji" onClick={onConnectWithDeviceClick} className="connection-method-link">
-                    <span>Connect with another device</span>
-                  </button>
-                </div>
-                <div className="connection-method passphrase-method">
-                  <h2 className="connection-method-title">Passphrase</h2>
-                  <p className="connection-method-text">If you have lost all your devices, recover the access to your account.</p>
-                  <button id="recover" onClick={onConnectWithPassphraseClick} className="connection-method-link">
-                    <span>Connect with passphrase</span>
-                  </button>
-                </div>
+    <div className={useClassFor('connection')}>
+      <div className={classForComponent('connection-wrapper')}>
+        <h1 className={classForComponent('connection-title')}>{(theme === 'unilogin') ? 'Connect account' : 'Connect with another device'}</h1>
```

Comment:
I would change the copy for all styles to `Connect account` instead of making two different texts.